python/ray/_private/worker.py

```diff
@@ -2816,18 +2856,49 @@ def method(self):
         @ray.remote(num_gpus=1, max_calls=1, num_returns=2)
         def f():
             return 1, 2
-        g = f.options(num_gpus=2)
+
+        f_with_2_gpus = f.options(num_gpus=2)
+        object_ref = f_with_2_gpus.remote()
+        assert ray.get(object_ref) == (1, 2)
+
 
         @ray.remote(num_cpus=2, resources={"CustomResource": 1})
         class Foo:
             def method(self):
                 return 1
-        Bar = Foo.options(num_cpus=1, resources=None)
+
+        Foo_with_no_resources = Foo.options(num_cpus=1, resources=None)
+        foo_actor = Foo_with_no_resources.remote()
+        assert ray.get(foo_actor.method.remote()) == 1
+
 
     Running remote actors will be terminated when the actor handle to them
     in Python is deleted, which will cause them to complete any outstanding
-    work and then shut down. If you want to kill them immediately, you can
-    also call ``ray.kill(actor)``.
+    work and then shut down. If you only have 1 reference to an actor handle,
+    calling ``del actor`` *could* trigger actor deletion. Note that your program
+    may have multiple references to the same ActorHandle, and actor termination
+    will not occur until the reference count goes to 0. See the Python
+    documentation for more context about object deletion.
+    https://docs.python.org/3.9/reference/datamodel.html#object.__del__
+
+    If you want to kill actors immediately, you can also call ``ray.kill(actor)``.
+
+    .. tip::
+        Avoid repeatedly passing in large arguments to remote task or method calls.
```

Comment:
Can we convert it to a Ray anti-pattern and link here?

Or you can add a TODO and I'll do it.

Reply:
done